episodes/chapter4.md

```diff
@@ -0,0 +1,242 @@
+    - The permissions of the user are defined by the assigned **rules**.
+- **Node**: a server that can execute **tasks**.
+- **Task**: a task that is executed by one or more **nodes**.
+    - A task should produce an algorithm run for each organization involved in the task. The results are part of such an algorithm run.
+    - A task can be part of a **study**.
+
+The following diagram is simplified relations between these entities. A `1-n` relationship means that the entity on the left side of the relationship can have multiple entities on the right side. For instance, a single organization can have multiple Vantage6 users, but a single user always belongs to one organization. There is one `0-n` relationship between roles and organizations, since a role can be coupled to an organization, but it doesn’t have to be. An `n-n` relationship is a many-to-many relationship: for instance, a collaboration can contain multiple organizations, and an organization can participate in multiple collaborations.
+
+![Vantage6 relations between entities](fig/v6_entities.png)
+
+### Where are the entities in the UI?
+
+After logging in to the Vantage6 UI, you will see the start page.
+
+![Vantage6 UI start page](fig/ui_start_page.png)
+
+There are some collbarations displayed on the start page. Clicking one of the collaborations will show the tasks of that collaboration.
+
+![Vantage6 UI tasks page](fig/ui_task_page.png)
+
+The start page also contains a button `Adaministration` in the top right corner. Clicking on this button will redirect you to the administration page.
+
+In the administration page, you can manage the entities of Vantage6. The entities are divided into tabs: `Organizations`, `Collaborations`, `Roles`, `Users`, and `Nodes`. You can click on an entity to see more details or to edit the entity. We will get back to this later in more detail.
+
+![Vantage6 UI administration page](fig/ui_admin_page.png)
+
+::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge 1: Get familiar with the Vanatge6 UI
+
+Log in to the Vantage6 UI and navigate to the `Administration` page. Familiarize yourself with the entities in the administration page.
+
+Can you find the `Organizations`, `Collaborations`, `Roles`, `Users`, and `Nodes` tabs?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Vantage6 permission system
+
+Vantage6 uses a permission system to control who can do what in the system. The permission system is based on **roles**, which are collections of rules that define the permissions of a user. A user can have multiple roles, and the permissions of the user are defined by the assigned **rules**.
+
+The permission rules define what each entity is allowed to do, based on the operation (view, create, edit, delete, send, receive), the scope (own,  organization, collaboration, global), and the resource (e.g. users, orgarnizations). Users can be assigned anywhere between zero and all of these permission rules.
+
```

Comment:
Add an example here: "Having the rule with operation 'create', scope Y, resource Z will allow you to ..."